null_safety/calculate_lix/pubspec.lock

```diff
@@ -0,0 +1,14 @@
+# Generated by pub
```

Comment:
I don't think we want to pin people in the lock file, but instead specify our deps in the pubspec.yaml file (otherwise we could pin people  to older deps than we intend).

Reply:
Oh, yes, I guess the dep is pinned in `pubspec.yaml` so we don't need the lock file? Still, I thought the guidance for apps was to checkin `.lock`?

Reply:
Here, I think we'll be pining customers to unintentionally older versions.

Reply:
But is that not required as this is the single version of collection we know works with null safety?